### DISCUSSION TITLE
Licensing

### DISCUSSION THREAD

**Discussion by vedxp** [vuejs/vitepress#1407] [Q&A] [answered]
Can I license my site built with vitepress under GNU GPL 3?

Or I've to license under MIT only?

**Comment by brc-dd** [comment, marked as answer]
> Can I license my site built with vitepress under GNU GPL 3?

Yeah do whatever you want.

**Reply by vedxp** [reply]
That means I can use any MIT licensed project and license my version of that project under GPL?

Thank you sir!

**Reply by brc-dd** [reply]
It's your code that will be licensed as GPL. Our source code will still be present as MIT. If your code is comprised of our code (e.g. you're bundling or modifying/using our code), then you need to distribute a copy of our license along with yours.

If you're just using VitePress to build your site, we don't need any attribution. You can license it whatever you want.

PS: ask these questions on https://opensource.stackexchange.com instead.

**Reply by vedxp** [reply]
Oh, thank you so much for the explanation!

All doubts cleared now 🙃
